index.js

```diff
@@ -0,0 +1,25 @@
+// alert('hey how ya going, mate?!');
+
+const boxerStorage = JSON.parse(localStorage.getItem('boxers'));
+const table = document.querySelector('#boxer-list');
+
+
+(function printBoxers() {
+  // debugger;
+  boxerStorage.forEach(e => {
+    const tr = document.createElement('tr');
+    table.appendChild(tr);
+    const boxerData = [
+      [`Name: ${e.firstName} ${e.lastName}`],
+      [`Wins: ${e.wins} `],
+      [` Losses: ${e.losses} `]
```

Comment:
Why did you make these arrays?

Reply:
I cannot answer this. I have undone this unnecessary array and the subsequent spread